### PR TITLE
April1: Backwards mode 🙃 

### PR DESCRIPTION
**Pull request by canadianveggie**
To test set puzzleId = 50

Screen Shot 2022-03-31 at 9 59 18 AM

### CODE REVIEW

**Comment by canadianveggie** [comment]
I've tested this a bit. It's unique but a little annoying. Not sure this is what we want.

File: index.html [hunk 1]
```diff
@@ -64,7 +64,10 @@
             <i class="material-icons">share</i>
           </a>
         </div>
-        <div class="title">Spellie</div>
+        <div class="title">
+          Spellie
+          <span x-cloak x-show="puzzleId === 50">🙃</span>
```
Comment (wachunga) [inline index.html:69]: Low budget tooltip?

```suggestion
          <span x-cloak x-show="puzzleId === 50" title="April Fools!">🙃</span>
```

**Review by wachunga — APPROVED** [review]

**Comment by wachunga** [comment]
oops @canadianveggie that title won't work because of `pointer-events: none` on the parent

**Comment by canadianveggie** [comment]
I haven't merged yet

**Pull request merged**